roles/contiv_network/files/v2plugin.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -euxo pipefail
+set -e
```

Comment:
Why this change?  We generally want to keep these things enabled to make our scripts harder to mess up 😄